src/dcat-us/dataset-formatter.ts

```diff
@@ -18,6 +18,7 @@ export function formatDcatDataset (hubDataset: HubDatasetAttributes, siteUrl: st
   } as DatasetResource);
   const { relative: relativePath } = getContentSiteUrls(content, siteModel);
   const landingPage = siteUrl.startsWith('https://') ? siteUrl + relativePath : `https://${siteUrl}${relativePath}`;
+  const downloadLink = siteUrl.startsWith('https://') ? `${siteUrl}/datasets/${hubDataset.slug}` : `https://${siteUrl}/datasets/${hubDataset.slug}`;
```

Comment:
Change this to use `hubDataset.identifier`. It returns the slug if available and the hubId on the off chance the slug doesn't exist. It's a little more robust and will never be undefined 👍

Reply:
Done thanks for the heads up